doc/classes/ResourceImporterScene.xml

```diff
@@ -62,6 +62,9 @@
 		<member name="nodes/root_type" type="String" setter="" getter="" default="&quot;&quot;">
 			Override for the root node type. If empty, the root node will use what the scene specifies, or [Node3D] if the scene does not specify a root type. Using a node type that inherits from [Node3D] is recommended. Otherwise, you'll lose the ability to position the node directly in the 3D editor.
 		</member>
+		<member name="nodes/use_node_type_customization" type="bool" setter="" getter="" default="true">
+			If [code]true[/code], enables the use of node type customization. Disable when automatic transformations based on naming conventions are not desired. See [url=$DOCS_URL/tutorials/assets_pipeline/importing_3d_scenes/node_type_customization.html]ImportNode type customization using name suffixes[/url] for more information.
+		</member>
```

Comment:
This is not the best name for this feature. We don't want to provide the user with an option to disable all advanced node types in the importer. As I mentioned above, if the file contains explicit data we should always read that data. I would limit this feature purely to the implicit node name type suffixes.

```suggestion
		<member name="nodes/allow_node_type_suffixes" type="bool" setter="" getter="" default="true">
			If [code]true[/code], enables the use of node name suffixes to control the generated node type. Disable when node names changing the node types are not desired. See [url=$DOCS_URL/tutorials/assets_pipeline/importing_3d_scenes/node_type_customization.html]ImportNode type customization using name suffixes[/url] for more information.
		</member>
```